test/test_clip2.jl

```diff
@@ -0,0 +1,27 @@
+# CLIP
+# Get CLIP model config from huggingface
+using Revise
+using Transformers
+using TextEncodeBase
+using Transformers.HuggingFace
+
+
+clip_model_name = "openai/clip-vit-large-patch14"
+clip_config = Transformers.load_config(clip_model_name)
+embeddings = HuggingFace.HGFCLIPTextEmbeddings(clip_config.text_config)
+state = Transformers.load_state(clip_model_name)
+HuggingFace.load_state!(embeddings, state.text_model.embeddings)
+
+tkr = hgf"openai/clip-vit-large-patch14:tokenizer"
+
+# for Diffusers, token vectors must return 77 element vector
+tkr = Transformers.Basic.TransformerTextEncoder(tkr) do enc
```

Comment:
```suggestion
tkr = Transformers.GenerativePreTrain.GPT2TextEncoder(tkr) do enc
```

Reply:
I'm not familiar with the Diffusers, but having the encoder return vector with 77 tokens means for short input most of the embedding is just the embedding of padding token. Is that an expected behavior?

Reply:
Yes, it is the expected behavior.